packages/trpc/server/routers/viewer/eventTypes/duplicate.handler.ts

```diff
@@ -15,6 +17,13 @@ type DuplicateOptions = {
   input: TDuplicateInputSchema;
 };
 
+export const generateHashedLink = (id: number) => {
```

Comment:
If we search `const generateHashedLink` it is duplicated across 3 files

<img width="1803" alt="Screenshot 2024-02-06 at 11 37 13" src="https://github.com/calcom/cal.com/assets/42170848/7913c89c-fd89-4e77-9b20-5052de313da1">

It would be good to re-use `generateHashedLink` - the copies exist across "apps/web", "packages/features" and "packages/trpc" and all of them import from "@calcom/lib" package ("packages/lib"), so I think it would be good to move `generateHashedLink` there and then re-use across these three packages (it is used in 3 files in total)

Reply:
absolutely